.storybook/components/DesignTokens/Tier1/Colors.js

```diff
@@ -27,13 +27,3 @@ export class Tier1Colors extends Component {
     );
   }
 }
-
-function filterTokens(prefix) {
-  return Object.entries(tokens)
-    .filter(([name]) => name.startsWith(prefix))
-    .map(([name, { value, comment }]) => ({
```

Comment:
this was expecting the wrong structure -- it should have been parsing for `[name, value]`. moved this into its own `util` function so I wouldn't have to fix it everywhere